web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/ChannelController.java

```diff
@@ -462,6 +461,18 @@ public WrapperEntitiesOperations updateWrapperChannelDetailsByOpt(@RequestBody
         return createdWrapperEntities;
     }
 
+    @GetMapping(  value = "/wfespplugins",  produces = {MediaType.APPLICATION_JSON_VALUE})
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.api.channels.updateWrapperChannelDetailsByOpt}")
+    public ResponseEntity<WfespPluginConfs> getWfespPlugins() {
+        log.trace("getWfespPlugins start");
+        String xRequestId = UUID.randomUUID().toString();
```

Comment:
If the Frontend passes an `xRequestId`, it may be used to trace the whole call.

TIPS: to handle the requestId in a smart way you can use [this approach](https://github.com/pagopa/pagopa-api-config/blob/main/src/main/java/it/gov/pagopa/apiconfig/core/config/RequestFilter.java#L45-L52) and you can retrive it with `MDC.get("requestId")` ([example](https://github.com/pagopa/pagopa-afm-utils/blob/main/src/main/java/it/gov/pagopa/afm/utils/config/FeignConfig.java#L18))

Is possible to [customize](https://github.com/pagopa/pagopa-api-config/blob/main/src/main/resources/logback-spring.xml#L9) your logging to add automatically the request